src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -12626,7 +12628,10 @@ void VRFTerminalUnitEquipment::ControlVRF_FluidTCtrl(EnergyPlusData &state,
         if (NoCompOutput <= QZnReq) return;
     } else if (VRFCoolingMode || HRCoolingMode) {
         // IF the system is in cooling mode and/or the terminal unit requests cooling
-        if (NoCompOutput <= QZnReq) return;
+        if (NoCompOutput <= QZnReq || QZnReq >= DataHVACGlobals::SmallLoad) {
```

Comment:
Shouldn't this be ... || QZnReq > -SmallLoad ?

Reply:
Standard practice is to clean up the functions that get modified. First, of course, make the changes and let someone review them and make comments. Then go back and delete using statements, use local declarations. E.g., delete 

int VRFCond;

and instead use:

int VRFCond = this->VRFSysNum;

delete the na comments in the function descrption, etc.

Reply:
> Shouldn't this be ... || QZnReq > -SmallLoad ?

In this case I am interested in the heating load only. Also if the heating load is below the threshold of 5W it should be ignored. If it makes a difference, I will change it to `-SmallLoad`.